test/functional/feature_pruning.py

```diff
@@ -79,11 +78,8 @@ def test_height_min(self):
         for i in range(25):
             mine_large_block(self.nodes[0], self.utxo_cache_0)
 
-        waitstart = time.time()
-        while os.path.isfile(self.prunedir+"blk00000.dat"):
-            time.sleep(0.1)
-            if time.time() - waitstart > 30:
-                raise AssertionError("blk00000.dat not pruned when it should be")
+        # Wait for blk00000.dat to be pruned
+        wait_until(lambda: not os.path.isfile(self.prunedir+"blk00000.dat"), timeout=30)
```

Comment:
I understand this is just preserving the previous code, but this should probably be updated to use `os.path.join` like:
```
# Before 
os.path.isfile(self.prunedir+"blk00000.dat")  
# After
os.path.isfile(os.path.join(self.prunedir, "blk00000.dat"))
```

There are several instances of this unrelated to this PR, so perhaps it can just be done in a follow up PR.

Reply:
Thanks there were a few such cases in this file, but not elsewhere.